wasm/wazeroir/interpreter.go

```diff
@@ -547,9 +520,13 @@ func (it *interpreter) callHostFunc(f *interpreterFunction, _ ...uint64) {
 
 func (it *interpreter) callNativeFunc(f *interpreterFunction) {
 	frame := &interpreterFrame{f: f}
-	memoryInst := frame.f.funcInstance.ModuleInstance.Memory
-	globals := frame.f.funcInstance.ModuleInstance.Globals
-	tables := f.funcInstance.ModuleInstance.Tables
+	moduleInst := f.funcInstance.ModuleInstance
+	memoryInst := moduleInst.Memory
+	globals := moduleInst.Globals
+	var table *wasm.TableInstance
+	if len(moduleInst.Tables) > 0 {
+		table = moduleInst.Tables[0] // We don't support yet.
```

Comment:
change to not our fault :D

```suggestion
		table = moduleInst.Tables[0] // WebAssembly 1.0 (MVP) defines at most one table
```